src/status_im/ui/screens/signing/views.cljs

```diff
@@ -182,7 +182,7 @@
       :error amount-error
       :accessories [[react/view {:style {:flex-direction :row}}
                      [react/nested-text {:style {:color colors/gray}}
-                      [{:style {:color colors/black}} (utils/format-decimals amount 6)]
+                      [{:style {:color colors/black}} (str amount)]
```

Comment:
we need to keep this

Reply:
this is by design

Reply:
so, it's okay if it shows `0.00000` if the number is very small?

Reply:
mhmh, thats not expected for sure :) @errorists any ideas? we did this with Andrey, but not sure why we missed this case

Reply:
![Screenshot from 2020-04-16 12-03-41](https://user-images.githubusercontent.com/1106587/79479432-bedd6580-7fda-11ea-9556-fe2e65804e3d.png)
For reference, this is how it currently looks in this PR,   notice how the amount at the top contains all the decimals but the amount+fee below only shows the first 6